src/settings/index.js

```diff
@@ -21,7 +21,9 @@ const settings = {
     borrowingManager: '0xD1760AA0FCD9e64bA4ea43399Ad789CFd63C7809',
     epochsManager: '0x15Ff10fCc8A1a50bFbE07847A22664801eA79E0f',
     registrationManager: '0x906B067e392e2c5f9E4f101f36C0b8CdA4885EBf',
-    feesManager: '0xDf951d2061b12922BFbF22cb17B17f3b39183570'
+    feesManager: '0xDf951d2061b12922BFbF22cb17B17f3b39183570',
+    pNetworkAssociationGnosisSafeAddress: '0xf1f6568a76559d85cF68E6597fA587544184dD46',
+    pNetworkV2VaultAddess: '0xe396757ec7e6ac7c8e5abe7285dde47b98f22db8'
```

Comment:
We can have other vaults (i.e. on bsc) so better differentiate them in the naming
```suggestion
    pNetworkV2EthereumVaultAddess: '0xe396757ec7e6ac7c8e5abe7285dde47b98f22db8'
```